CNV-62285: Move vTPM note to docs (#103007)
openshift/openshift-docs: modules/virt-adding-vtpm-to-vm.adoc

(VM) allows you to run a VM created from a Windows 11 image without a physical
TPM device. A vTPM device also stores secrets for that VM.

[IMPORTANT]
====

> When you add a virtual Trusted Platform Module (vTPM) device to a Windows VM, it is important to make the vTPM device persistent. The BitLocker Drive is encrypted successfully and the encryption system check passes even if the vTPM device is not persistent. If the vTPM device is not persistent, it is discarded on shutdown.

Reviewer: ```suggestion
When you add a vTPM device to a Windows VM, you must make that device persistent. If the vTPM device is not persistent, the BitLocker Drive encryption system check still passes and the vTPM device is discarded on shutdown.
```

Reviewer 2: I like the original "it is important", rather than "must". Many recent versions of Windows refuse to install if a TPM is not present, regardless of BitLocker settings. However I believe all versions of Windows still let you keep BitLocker disabled.
If I was a CNV user with Windows VMs, I would actually give those VMs a non-persistent TPM just to pass the check, but I wouldn't enable persistence/BitLocker, to avoid the associated costs (storage for persistence and CPU for BitLocker), and just rely on a good CSI to keep my VM disks secure.
That being said, maybe we want to play it safe and say we don't support the creation of Windows VMs without persistent TPM, but that decision is above my paygrade.
